homeassistant/components/blebox/binary_sensor.py

```diff
@@ -0,0 +1,38 @@
+"""BleBox sensor entities."""
```

Comment:
This addition doesn't need to go in the same PR as is removing the air quality platform.

Please move to a separate PR.